### PR TITLE
Publish October 2022 RubyGems Update

### PR DESCRIPTION
**Pull request by lauragift21**
Draft of RubyGems Update for October.

Hi, @deivid-rodriguez want to help review the RubyGems section to ensure I covered the important updates.

### CODE REVIEW

File: _posts/2022-11-10-october-rubygems-updates.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+The following improvements and fixes are also included in these releases (see the changelog for more information):
+
+- made the `util/rubocop` script use `in-tree` Bundler - [#5979](https://github.com/rubygems/rubygems/pull/5979).
+- added extra resolver spec groups for Ruby platform when required - [#5698](https://github.com/rubygems/rubygems/pull/5698).
+- added `SHA256` in test certificates - [#5982](https://github.com/rubygems/rubygems/pull/5982).
+- GitHub DSL documentation was updated to reflect that it uses the HTTPS protocol under the hood instead of the git protocol. - [#5993](https://github.com/rubygems/rubygems/pull/5993).
+- allow JRuby to pass keywords to `Kernel#warn` - [#6002](https://github.com/rubygems/rubygems/pull/6002).
+- specified all GitHub sources use the HTTPS secure protocol by default - [#6026](https://github.com/rubygems/rubygems/pull/6026).
```
Comment (deivid-rodriguez) [inline _posts/2022-11-10-october-rubygems-updates.md:21]: I think this one can be unified with the bullet point of "GitHub DSL documentation was updated ....". I'd say

* Unify source code and documentation to always use HTTPS under the hood with dealing with GitHub sources. - [#5993](https://github.com/rubygems/rubygems/pull/5993) and [#6026](https://github.com/rubygems/rubygems/pull/6026).

File: _posts/2022-11-10-october-rubygems-updates.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
```
Comment (deivid-rodriguez) [inline _posts/2022-11-10-october-rubygems-updates.md:22]: I would also add the following bullet points:

* Several issues with `Gem::Platform` handling were fixed in `musl` platforms ([#5915](https://github.com/rubygems/rubygems/pull/5915)), in `arm` platforms with `wabi` modifiers ([#5957](https://github.com/rubygems/rubygems/pull/5957)), and to properly deal with string parameters when comparing ([#5939](https://github.com/rubygems/rubygems/pull/5939)).
* We improved handling of permanent redirect responses when pushing gems. - [#5931](https://github.com/rubygems/rubygems/pull/5931).
* We fixed an obscure issue affecting file extraction of some specific `.gem` packages. - [5906](https://github.com/rubygems/rubygems/pull/5906/files)
* We migrated the GitLab CI template generated by `bundle gem` to be the one now recommended by GitLab.

File: _posts/2022-11-10-october-rubygems-updates.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in October.
+
+## RubyGems News
+This month in RubyGems, we released new versions of RubyGems [3.3.23](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3323--2022-10-05), [3.3.24](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3324--2022-10-17) and Bundler [2.3.23](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2323-october-5-2022), [2.3.24](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2324-october-17-2022).
+
+
+The following improvements and fixes are also included in these releases (see the changelog for more information):
+
+- made the `util/rubocop` script use `in-tree` Bundler - [#5979](https://github.com/rubygems/rubygems/pull/5979).
```
Comment (deivid-rodriguez) [inline _posts/2022-11-10-october-rubygems-updates.md:16]: I would reword this one to mention it's only a development environment improvement that does not affect end users.

```
* We had a small development environment contributed to make our `util/rubocop` script use "in-tree" Bundler - [#5979](https://github.com/rubygems/rubygems/pull/5979).
```

File: _posts/2022-11-10-october-rubygems-updates.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- added extra resolver spec groups for Ruby platform when required - [#5698](https://github.com/rubygems/rubygems/pull/5698).
```
Comment (deivid-rodriguez) [inline _posts/2022-11-10-october-rubygems-updates.md:17]: Let's reword this one too

```
* Improved resolution performance and correctness by adding resolver spec groups for Ruby platform only when necessary. This is in preparation for the upcoming migration to Pub Grub - [#5698](https://github.com/rubygems/rubygems/pull/5698).

**Review by deivid-rodriguez — COMMENTED** [review]
I added some suggestions!

This month we got quite a few contributions from external people. I wonder whether we should start shouting out to them.

**Comment by lauragift21** [comment]
> I added some suggestions!
> 
> This month, we got quite a few contributions from external people. I wonder whether we should start shouting out to them.

Thanks for the suggestions! @deivid-rodriguez, yes, we can start including contributions from external people. If I get a note of what contributions are marked as external, I can add a section to the end of the post thanking the contributors.
